hubresolver/pkg/hub/params.go

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2022 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hub
+
+// DefaultHubURL is de default url for the Tekton hub api
+const DefaultHubURL = "https://api.hub.tekton.dev/v1/resource/Tekton/%s/%s/%s/yaml"
+
+// YamlEndpoint is the suffix for a private custom hub instance
+const YamlEndpoint = "v1/resource/Tekton/%s/%s/%s/yaml"
```

Comment:
the catalog field mentioned above will be substituted here
```suggestion
// DefaultHubURL is de default url for the Tekton hub api
const DefaultHubURL = "https://api.hub.tekton.dev/v1/resource/%s/%s/%s/%s/yaml"

// YamlEndpoint is the suffix for a private custom hub instance
const YamlEndpoint = "v1/resource/%s/%s/%s/%s/yaml"
```

Reply:
and we can support defaulting to the "tekton" catalog - cc @chitrangpatel